index.js

```diff
@@ -1301,7 +1301,7 @@
   //. Nil
   //. ```
   function empty(typeRep) {
-    return Monoid.methods.empty(typeRep)();
+    return Monoid.functions.empty(typeRep)();
```

Comment:
You may not approve of this change, since suddenly function definitions need to know where the implementation lives. If that's a problem we can simply add something along the lines of...:

```js
typeClass.mixed = typeClass.functions.concat(typeClass.methods);
```

...at the end of `$`, and use the `mixed` property to access implementations.